### PR TITLE
Studio update images

### PR DESCRIPTION
**Pull request by tapadipti**
- Update outdated screenshots in the [explore-ml-experiments page](https://dvc.org/doc/studio/user-guide/projects-and-experiments/explore-ml-experiments).
- Update some of the text in this page to better fit the updated screenshots.

Completes https://github.com/iterative/dvc.org/issues/4679 and https://github.com/iterative/studio/issues/4770

Required png and gif files have been updated in the static repo in [this commit](https://github.com/iterative/static/commit/5e9f4eee257b550e2021f7907c5405e1e9fed348).

### CODE REVIEW

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:63]: this is a very confusing behavior / name for the filter, and the doc explanation here doesn't help the difference stand out (i.e. I'm not even sure I get it after reading a few times 😄  )
Suggestion (IIUC):

> On the other hand, the `Commits on branch` filter displays all branches that  contain commits from the specified branch.

->

> On the other hand, the `Commits on branch` filter will also display branches that contain any commits **in common** with the specified branch.

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:67]: will it include all of `feature-branch-1`'s commits? experiments? just the common commit with main? please clarify.
What would really help here is to explain why this exist - what's the scenario? e.g. filter by main/develop branch and see all of the feature branches that spawned out of it

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:83]:
```suggestion
dropdown:
```

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:73]:
```suggestion
Select the columns you want to display and hide the rest:
```

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:128]: I would mention that this is the behavior for numerical values (how would other values like text be treated?)

Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:128]:
```suggestion
toggle between these two options, use the `Delta mode` button:
```
There might be more instances.
Maybe it's a stylistic choice, but it's very weird for me to see a sentence end in a period (`.`) if it is explicitly demonstrated by an image right underneath it (expect a `:`)

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:155]:
```suggestion
On smaller screens, the buttons might appear without text labels:
```

**Review by omesser — CHANGES_REQUESTED** [review]
Hey @tapadipti  - Left a few comments! hope this helps

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:110]: See [this comment](https://github.com/iterative/dvc.org/pull/4758#discussion_r1293408595).

**Comment by tapadipti** [comment]
@omesser Thanks for your review. I've replied to your comments - ptal.

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (mvshmakov) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:155]: > Can you pls confirm this?

It works for metrics, models and files, e.g.,

Screenshot 2023-08-14 at 21 40 49

> Also, why doesn't Delta mode work on hyper-parameters - is it intended behavior or a bug?

Neither, I believe. Historically, the params were just strings, and at some point we started to distinguish between different types. We've supported all types, but just forgot to implement delta mode for them. Feel free to file an issue, no tech blockers exist behind the numerical params delta mode support.

Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:155]: I've updated the text [here](https://github.com/iterative/dvc.org/pull/4758/commits/11b0bd52b4edfe3e5a74f40be650e5307807db8a).

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (jellebouwman) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:39]: Maybe note that the merged branch will only show the commits that are not present in `main`?

**Review by jellebouwman — COMMENTED** [review]
Thank you @tapadipti for adding this!

**Review by jellebouwman — APPROVED** [review]
Thanks!

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:156]: Not feeling strongly (this should not block this PR to be clear), but it does look weird IMO (going from `.` to an image demonstrating the above description).
I would suggest on aligning on `:`  - not necessarily in this PR

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:50]: Wanted to suggest some rewording, my key goals were:
- not refering to the branch as "merged" if its tip is not merged.
- clarifying that this example can be projected on any case where some "trunk" branch accumulates changes from working/feature branches.
- using the terms "active" and "inactive" to illustrate the perceived importance of the branch to the user.

I ended up with that:

> 
> For illustration purposes, we'll assume a Git branch, named `my-exp`, which includes some unique commits, was merged into the `main` branch (pull request target). In this example `main` is the "trunk" in the team, where active work is being merged to. In reality, this can also any branch that the team chooses of course, like `master, `dev`, `develop`, or some integration/release branch. Often this is also the default branch for the repository.
> 
> After a PR from a Git branch is merged, two possibilities exist:
> 
> 1. `my-exp` branch is still active - the user continues to push commits to it after it was merged to `main` on some pull request commit, which is no longer its tip. Since the branch contains new unique commits, the project table will display both the target branch (`main`) and the feature branch (`my-exp`) separately. Here, `my-exp` will only show the new commits that are not part of `main`.
> 
> 2. `my-exp` is inactive - No new commits were pushed to `my-exp` after it was merged to `main`. In this scenario, Iterative Studio considers `my-exp` branch as **"nested branch"** and does not display it as a separate branch in the project table by default. This helps to keep the project table concise and reduce clutter that can accumulate over time when inactive branches are not cleaned from the project repository. Those "inactive branches" usually carry no new information for the purpose of managing experiments in Studio. If you would like to force display such an "inactive branch", use the `"Commits on branch"=my-exp` display filter.

wdyt @tapadipti ?
Feel free to adopt in whole/parts or drop

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:156]: Create https://github.com/iterative/dvc.org/issues/4787

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:82]:
```suggestion
    The `Branch` filter displays only the specified branch and its commits.
```

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:85]: This sentence means that `Commits on branch = main` will also cause `feature-branch-1` and all of its commits to be shown. is this the case? or you have to do `Commits on branch = feature-branch-1` to show an merged/inactive `feature-branch-1` ?

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:90]:
```suggestion
    `feature-branch-1` is "nested" within `main`), commits from this branch are
```

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (omesser) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:92]: 😖 Sorry, this is still very confusing to read.
So, IIUC - if `feature-branch-1` is nested in `main` (=it is inactive, no new unique commits), AND I use `Commits on branch = feature-branch-1` all of `feature-branch-1` commits will appear within `main` and I'll have an empty `feature-branch-1` row?

**Review by omesser — APPROVED** [review]
I find the branch filter text still very difficult to follow and left some comments and a suggestion to address at least a part of it (the new nested branches content).
but approving this nonetheless to unblock the PR

**Comment by shcheklein** [comment]
@tapadipti what is the status of this?

**Comment by tapadipti** [comment]
> @tapadipti what is the status of this?

@shcheklein I'll review the last comments by Oded and then merge this.

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md]: Thanks Oded. I've adopted this partially (in [this commit](https://github.com/iterative/dvc.org/pull/4758/commits/40e6b455f1d0e260370c22457b0051672c82d744)), and I think it's much clearer now.

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md]: [Done](https://github.com/iterative/dvc.org/pull/4758/commits/40e6b455f1d0e260370c22457b0051672c82d744)

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md]: [Revised](https://github.com/iterative/dvc.org/pull/4758/commits/40e6b455f1d0e260370c22457b0051672c82d744)

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md]: [Revised](https://github.com/iterative/dvc.org/pull/4758/commits/40e6b455f1d0e260370c22457b0051672c82d744). There'll be no `feature-branch-1` row.

**Comment by tapadipti** [comment]
@omesser Thanks for your suggested text for the nested branches and commits on branch filter. I think it makes that section much clearer. I'll merge this PR now. But feel free to open a new issue if you think it needs some more revision.

**Comment by tapadipti** [comment]
The formatting check is failing for this PR. But it's complaining about a file that this PR does not touch (`content/docs/user-guide/integrations/sagemaker.md`). Can I ignore this? Who can suggest - is it @yathomasi? 


image

**Comment by yathomasi** [comment]
@tapadipti Could you please run `yarn fix-all` and see if it fixes the file and commit and push the change would help ?

**Comment by tapadipti** [comment]
@yathomasi I ran `yarn fix-all` and now all these files have changes.

```
content/docs/user-guide/integrations/sagemaker.md
src/@dvcorg/gatsby-theme-iterative/components/LayoutHeader/Nav/Popup/styles.module.css
src/@dvcorg/gatsby-theme-iterative/components/LayoutHeader/Nav/styles.module.css
src/@dvcorg/gatsby-theme-iterative/components/MainLayout/fonts.css
src/@dvcorg/gatsby-theme-iterative/data/styles.module.css
src/components/Community/Section/styles.module.css
src/components/DownloadButton/styles.module.css
src/components/Home/LandingHero/styles.module.css
src/components/Home/LogosSlider/styles.module.css
src/components/Home/WhatsNewModal/styles.module.css
src/components/PromoSection/styles.module.css
src/components/Support/Popover/styles.module.css
src/components/Support/styles.module.css
```
Should I push just `content/docs/user-guide/integrations/sagemaker.md`?

**Comment by yathomasi** [comment]
It should be okay to commit and push them all

**Comment by tapadipti** [comment]
@yathomasi I couldn't commit all - it failed with `stylelint --fix [FAILED]`. So I just committed the `sagemaker.md` file. But now I've reverted even that, coz it's causing the following code block

image

to appear as 

image

I could add `\` in that command, but if it's been fine so far, I think it'll be ok now as well.

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md
Comment (shcheklein) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:162]: I would always use `:`. It's a bug to my mind that people don't use `:` before images and a paragraph that explains it.

**Pull request merged**
